superbench/runner/runner.py

```diff
@@ -460,14 +462,19 @@ def run(self):
                             continue
                         with open(self._output_path / 'hostfile', 'r') as f:
                             host_list = f.read().splitlines()
-                        pattern_hostx = gen_traffic_pattern_host_group(host_list, mode.pattern)
-                        for host_groups in pattern_hostx:
-                            para_rc_list = Parallel(n_jobs=len(host_groups))(
-                                delayed(self._run_proc)
-                                (benchmark_name, mode, vars={
-                                    'proc_rank': 0,
-                                    'host_list': host_group,
-                                }) for host_group in host_groups
+                        host_groups = gen_traffic_pattern_host_groups(host_list, mode.pattern)
```

Comment:
If you don't need path, why do you support in utils?

Reply:
same with above.